src/main/java/tc/oc/pgm/commands/MatchCommands.java

```diff
@@ -28,18 +33,19 @@
   @Command(
       aliases = {"matchinfo", "match"},
       desc = "Shows information about the current match")
-  public static void matchInfo(CommandSender sender, Match match) {
+  public static void matchInfo(CommandSender sender, MatchPlayer player, Match match) {
     // indicates whether we have game information from the match yet
     boolean haveGameInfo =
         match.getPhase() == MatchPhase.RUNNING || match.getPhase() == MatchPhase.FINISHED;
 
     sender.sendMessage(
         StringUtils.dashedChatMessage(
```

Comment:
Can we modernize this to `ComponentUtils.horizontalLineHeading`?

Reply:
Thanks for the tip @TheMolkaPL! The header looks great now 😄